From 82853fde09708fe2d0719fc3fdc5db221a36b53e Mon Sep 17 00:00:00 2001
From: vdanix <vishwanathx.dani@intel.com>
Date: Tue, 7 Feb 2023 13:26:22 +0530
Subject: [PATCH] Remove user prompts from host setup

Changes:
- removed prompt to update QEMU
- removed prompt to update grub
- removed prompt for reboot

Tracked-On: OAM-105716
Signed-off-by: vdanix <vishwanathx.dani@intel.com>

diff --git a/scripts/setup_host.sh b/scripts/setup_host.sh
index 7223f17..81b1c9d 100755
--- a/scripts/setup_host.sh
+++ b/scripts/setup_host.sh
@@ -29,10 +29,6 @@ function error() {
 function ubu_changes_require(){
     echo "Please make sure your apt is working"
     echo "If you run the installation first time, reboot is required"
-    read -p "QEMU version will be replaced (it could be recovered by 'apt purge ^qemu, apt install qemu'), do you want to continue? [Y/n]" res
-    if [ x$res = xn ]; then
-        return 1
-    fi
     sudo apt install -y wget mtools ovmf dmidecode python3-usb python3-pyudev pulseaudio jq
 
     # Install libs for vm-manager
@@ -201,10 +197,6 @@ function ubu_enable_host_gvt(){
     if [[ ! `cat /etc/default/grub` =~ "i915.enable_guc="(0x)?0*"7" ]] &&
        [[ ! `cat /etc/default/grub` =~ "i915.enable_gvt=1" ]]; then
 
-        read -p "The grub entry in '/etc/default/grub' will be updated for enabling GVT-g and GVT-d, do you want to continue? [Y/n]" res
-        if [ x$res = xn ]; then
-            return
-        fi
         if [[ ! `cat /etc/default/grub` =~ "intel_iommu=on i915.force_probe=*" ]]; then
             sed -i "s/GRUB_CMDLINE_LINUX=\"/GRUB_CMDLINE_LINUX=\"intel_iommu=on i915.force_probe=* /g" /etc/default/grub
         fi
@@ -309,12 +301,7 @@ function check_sriov_setup() {
 
 function ask_reboot(){
     if [ $reboot_required -eq 1 ];then
-        read -p "Reboot is required, do you want to reboot it NOW? [y/N]" res
-        if [ x$res = xy ]; then
-            reboot
-        else
-            echo "Please reboot system later to take effect"
-        fi
+       echo "Please reboot system to take effect"
     fi
 }
 
-- 
2.39.1

